server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/server/ExecutorRequestHandler.java

```diff
@@ -74,8 +73,8 @@ public SuccessResponse<Object> process(HttpMethod httpMethod, String uri, String
                 Long jobId = Long.parseLong(matcher.group(1));
                 String logContent = executorBiz.getLog(jobId,
                         UrlUtils.getQueryParameterFirst(uri, "logType"),
-                        (maxLine == null ? LogUtils.MAX_LOG_LINE_COUNT : Long.parseLong(maxLine)),
-                        (maxSize == null ? LogUtils.MAX_LOG_BYTE_COUNT : Long.parseLong(maxSize)));
+                        (maxLine == null ? 10000L : Long.parseLong(maxLine)),
+                        (maxSize == null ? (long) (2 << 19) : Long.parseLong(maxSize)));
```

Comment:
why set default as`2 << 19`, hard to understand, and please extract constant variable to constant field

Reply:
Since netty can't get the properties configured inside the Spring Bean, a default value has been added just in case, and is now defined in LogUtils